Entities/ACrab.h

```diff
@@ -515,6 +515,18 @@ int FirearmActivationDelay() const;
 	void SetLimbPathPushForce(float force);
 
 
+	/// <summary>
+	/// Gets this ACrab's Stride Sound. Ownership is not transferred!
+	/// </summary>
+	/// <returns>The SoundContainer for this ACrab's Stride Sound.</returns>
+	SoundContainer* GetStrideSound() const { return m_StrideSound; }
+
+	/// <summary>
+	/// Sets this ACrab's Stride Sound.
+	/// </summary>
+	/// <param name="newSound">The new SoundContainer for this ACrab's Stride Sound</param>
```

Comment:
Method comments should end with a `.` cause otherwise they'll look awful in doxygen output (when we eventually start using that regularly)